rasa/core/channels/slack.py

```diff
@@ -297,13 +316,18 @@ async def process_message(
 
         if metadata is not None:
             output_channel = metadata.get("out_channel")
+            if self.use_threads:
+                thread_id = metadata.get("ts")
```

Comment:
@b-quachtran This line confuses me - shouldn't it be `metadata.get("thread_id") now? If I'm not mistaken, can you create a PR with a quick fix?

Reply:
And then we should probably also extend the tests to include a case that would have caught this (e.g. a test where we get a message event, post a reply to the same thread, and assert that everything went as expected).
If you have time, this could go into the PR with the fix, otherwise let's just create an issue to add such a test.